### ISSUE TITLE
Ukraine- GIS Request, Virtual Support/TDY

### ISSUE THREAD

**Issue by kkale25** [ICPI/Engagements#3]
Original email sent out via ICPI@state

Hello Ukraine Team (OGAC & in Country),
Please forward to relevant colleagues who are not on this email.

ICPI is updating its communication practices and we are reaching out to ensure that our colleagues are aware of an upcoming ICPI engagement request.

This email intends to open the lines of communication. Please see below for additional details.

Context:
Ukraine was asked to produce maps for COP. In an effort to fulfill this request the team recognized that capacity building in GIS needs to occur.  The Ukraine team reached out to ICPI recently with a request for support on GIS capacity building. After review, the ICPI leads have approved the request to engage with Ukraine around GIS support. The ICPI GIS workstream has taken over the request and is discussing the scope.

Status:

* ICPI Leads Approved 10/30
* Underway: discussion of scope between ICPI workstream POC and Ukraine POCs

Title:     ICPI GIS Support to Ukraine
Completion Date: TBD

Scope of Current Request:
  *   The objective of the project is to enable the DRC country team (both SI and program staff) to better use PEPFAR data to conduct focused monitoring of PEPFAR-supported programs and partners in a way that directly and concretely facilitates action-i.e., the identification and remediation of gaps in program achievement or quality through partner management.

Next Steps:
* Field Team : Interagency teams review and comment
* GIS Workstream:
        o   The GIS workstream will negotiate the scope and details and final product that will come out of this engagement.
       o   Once updated, the ICPI and Ukraine GIS team will send a SOW for this project.
  *   ICPI: Continues to provide status updates on this engagement, including any documents and products coming out of this engagement.
  *   HQ: Chair, CL, SI Advisor, and ECT Management provide comments (if any)

Kindly respond with your comments, questions, or concerns.

**Comment by gsarfaty** [comment]
We have a few edits to make, @kkale25, and may have a bit more to info on staffing early next week. If possible to wait until Tues to send, that would be ideal. Let us know if that timeline does not work well for you, and if not we will at min. revise the draft for a Monday send out. @dp1984

**Comment by jb3436** [comment]
@gsarfaty - timeline sounds good. Thanks!

**Comment by dp1984** [comment]
Wanted to provide an update: we've had further conversations with Bob. We've outlined 3 options for him: 1. virtual training 2. Dustyn provide in-country training (after August) 3. Zhanar provide in-country training later in the year (returns from may leave July 3 and will be on limited travel).

We'll update above message with add'l information as well as include some of the above items that have been addressed.

**Comment by dp1984** [comment]
Bob Gern discussed the 3 proposed training options given be the ICPI/GIS Team with the Ukraine SI Team and discussed their needs and preferences regarding this TA.  They have decided that it would be best to wait a few more months until someone is available to come out here in-person rather than try to have us do virtual training for them. 

They would now like to discuss what dates after August would be possible. They have stated they  a discussion with Ukraine’s MoH regarding the timing of our PMDAT members going to the in-person PMDAT workshop either in late August or in early or mid-September.  So they would like us to work with them to set up dates around that.

**Comment by kkale25** [comment]
@dp1984 - any additional updates re: this request? Is this "on hold" at the moment? Would love to get some next steps going so we can re-activate and plan.

Thanks!

**Comment by jb3436** [comment]
@kzhanar and @jltobias 
Any updates to add here? The Ukraine TA plan has this engagement listed.

**Comment by kzhanar** [comment]
From recent communication with Bob Gern, Ukraine team requests an in-country training on DATIM GIS for USG staff and ArcGIS training for two staff members. 
There is no need in QGIS training for USG staff as QGIS software is not approved by IT in Ukraine to install on USAID computers.

**Comment by kkale25** [comment]
Notes from meeting on 11/2
Attendees: Jim, Kashmira, Zhanar, Travis, Ed Vallejo

- Request was made last year (Oct 30) from Bob to ICPI for a QGIS software training in Ukraine
          - At that time, we were trying to understand the type of analyses they'd need, create specific 
             maps/exercises, and understand their mapping needs 
          - Not enough time before COP to deliver training so provided remote GIS support by creating 
             maps for COP
           - They were still interested in GIS training; had some funds from an IP until April but since training 
              didn't happen, the funds expired
           - Bob's team learned in April that qGIS isn't allowed on certain computers so then changed their 
             request to ArcGIS training

Request now:
- Whole USG staff would like DATIM GIS training and ArcGIS training for 2 people
- How can we support this request and complete?
           - Training for DATIM GIS module --> could be done remotely (webinar series?); need 2 hrs so 
              traveling to Ukraine may not be best use of time
           -  For two ArcGIS users, we'll have to figure out how to best train them up; this type of training 
              depends on the user level (can be several days/weeks)
- What kinds of maps will they need to create? Still waiting on hearing from the country team the specific maps they need to make.
- Would have to create materials for the ArcGIS Training basically from scratch (have some resources but nothing concrete) 


Asks from Country Team:
- What level of users are the two folks that have the ArcGIS licenses? 
- What are the mapping needs? Need this info to customize training to specific training 
- Cyrillic fonts available for ArcGIS? If not...will English labels works for the country team? 
- Is this an interagency training or will it be CDC specific? 

Next steps:
- Talk to GIS cluster to figure out appropriate POCs
- GIS Cluster to have a call with Bob and country team to answer the above questions
- Let ICPI Coordination Office (Jasmine/Kashmira) know if additional support is needed

**Comment by jb3436** [comment]
@kzhanar providing ArcGIS Training Week of June 17
Training is a Training of Trainers - individual will then report out on results of training the rest of the team in Ukraine
TDY will not be necessary
